src/runners/order_watcher_service_runner.ts

```diff
@@ -1,19 +1,38 @@
 import { WSClient } from '@0x/mesh-rpc-client';
+import { Connection } from 'typeorm';
 
-import * as config from '../config';
-import { getDBConnectionAsync } from '../db_connection';
+import { getDefaultAppDependenciesAsync } from '../app';
+import * as defaultConfig from '../config';
 import { logger } from '../logger';
 import { OrderWatcherService } from '../services/order_watcher_service';
+import { providerUtils } from '../utils/provider_utils';
 
+if (require.main === module) {
+    (async () => {
+        const provider = providerUtils.createWeb3Provider(defaultConfig.ETHEREUM_RPC_URL);
+        const { connection, meshClient } = await getDefaultAppDependenciesAsync(provider, defaultConfig);
```

Comment:
thoughts on creating the provider in `getDefaultAppDependenciesAsync` ?

Reply:
I had that at first, but ran into trouble with the provider for Ganache :/